src/todo.ts

```diff
@@ -1,10 +1,9 @@
 import { Item } from './item';
-
 class TodoApp {
-  /**
-   * Allows the creation and maintenance of a simple todo list
-   */   
-    
+    /**
+     * Allows the creation and maintenance of a simple todo list
+     */
+
     public toDoList: Map<number, Item>;
```

Comment:
You can make this simpler by using an array.  Your ID at that point is simply the index of the array.  But there are pro/cons to this as well.

Reply:
The particular pro that made me use Map was that deleting and checking items is much faster with a map. The downside is listing all the items is slower and uses more lines of code

Reply:
Yep.  Also you could use sparse IDs with your implementation vs predictable IDs with my suggestion.  This was only a suggestion however... this implementation is just fins so long as you understand the trade-offs